### ISSUE TITLE
Make space before icon consistent across all buttons

### ISSUE THREAD

**Issue by kjfdebruin** [KennyMafabatho/bootstrap-tut#14]
![image](https://user-images.githubusercontent.com/10156093/95834469-4d8a2f80-0d3d-11eb-882a-a9bd9a3a2a11.png)
Either have a space before or no space before. Just be consistent

**Comment by KennyMafabatho** [comment]
Yeah, I changed buttons then wanted text to be grey, I changed that text-gray to text-secondary
